src/webportal/src/app/job-submission/components/docker-section.jsx

```diff
@@ -73,7 +78,7 @@ AuthTextFiled.propTypes = {
 const options = [
   {
     key: 'all',
-    text: 'all-in-one (image: ufoym/deepo:all)',
+    text: 'all-in-one+python3.6+gpu (image: ufoym/deepo:all-py36-cu100)',
     image: 'ufoym/deepo:all',
```

Comment:
Image should also be `ufoym/deepo:all-py36-cu100`

Reply:
Thanks, will fix